CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ All notable changes to this project are documented in this file.
  - The `mod` function was changed to always return a positive value (#6178)
  - Added `AccesssibleRole.list-item` and used it where relevant
  - Added `PointerEventButton.back` and `PointerEventButton.forward` (#6043)
+ - Added `PropertyAnimation.direction` property for controlling animation direction (#6261).
```

Comment:
Please put Remove the line from the CHANGELOG file and put it in a commit message instead.